uber/models.py

```diff
@@ -856,6 +861,7 @@ def model_mixin(cls, model):
 
 
 class Group(MagModel, TakesPaymentMixin):
+    search_key    = Column(UUID, default=lambda: str(uuid4()))
```

Comment:
...is this searching the group name (or for that matter, the attendee name) if it turns out not to be a QR code? I don't see that, but ... well, I'm old, so...